src/_blog/js-libp2p-v1.md

```diff
@@ -0,0 +1,139 @@
+
+### Reducing latency
+On average 80% improvement in the time it takes to establish a connection between two nodes.
+
+<div class="container" style="display:flex; column-gap:10px; justify-content: center; align-items: center;">
+    <figure>
+        <img src="../assets/js-libp2p-v1-latency-improvements.png" width="750">
+        <figcaption style="font-size:x-small;">
+            Latency of a connection between the two nodes.
+        </figcaption>
+    </figure>
+</div>
+
+### Increasing throughput
+We've been continously imnproving the throughput of js-libp2p, and currently js-libp2p has the highest throughput of any libp2p implementations averaging 2.06GB/s on both uploads and downloads!
+
+<div class="container" style="display:flex; column-gap:10px; justify-content: center; align-items: center;">
+    <figure>
+        <img src="../assets/js-libp2p-v1-throughput-improvements.png" width="750">
+        <figcaption style="font-size:x-small;">
+            Throughput of a connection between the two nodes.
+        </figcaption>
+    </figure>
+</div>
+
+
+### Reducing Dependencies
+
+We realized that the libp2p bundle size was unnecessarily big, and that we could reduce the bundle size by removing some of the modules that need not be included for many use cases, such as `fetch` , `UPnP`, `keychain` etc. This reduced the bundle size by over 40%! We still think there is room for even further reduction, and we will be working on this in the future.
+
+## Package Provenance
+
+There is no single answer to the problem of software supply chain integrity. However, there are a variety of techniques that can be used to mitigate the risk of supply chain attacks. One of the techniques we've implemented is package provenance. When you download libp2p from the npm registry you now have visibility into the process by which the source code was translated into the published artifact. You can verify the integrity of provenance attestations via the npm CLI. You can read more about package provenance [here](https://github.blog/2023-04-19-introducing-npm-package-provenance/)
+
+# Developer Experience Improvements 🌈
+
+We realized that there were a variety of pain points in the developer experience, and we wanted to make it as easy as possible to get started with libp2p and begin contributing. Here are some of the improvements we made:
```

Comment:
> We realized that

The same opening is used in the "Reducing dependencies" section.  Needs more variation.